plugins/scm.js

```diff
@@ -10,6 +10,15 @@ const jobName = Joi.reach(models.job.base, 'name').optional();
 const username = Joi.reach(models.user.base, 'username').required();
 const checkoutUrl = Joi.reach(models.pipeline.create, 'checkoutUrl').required();
 
+const GET_CHECKOUT_COMMAND = Joi.object().keys({
+    branch: Joi.string().required(),
+    host: Joi.string().required(),
+    org: Joi.string().required(),
+    repo: Joi.string().required(),
+    sha: Joi.string().required(),
```

Comment:
Because we will not checkout with `prRef` directly. We will fetch the pr and merge the pr's `sha` to the latest version of master branch.